client/packages/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.0.2-bin.zip
+networkTimeout=10000
```

Comment:
is this for building ? (if can't package fetching stops ?)

Reply:
yeah, the new capacitor requires gradle 8 or greater

Reply:
Sorry I was referring to `networkTimeout=10000` ?